MapboxDirections/MBRoute.swift

```diff
@@ -162,6 +162,20 @@ open class Route: NSObject, NSSecureCoding {
      */
     open let routeOptions: RouteOptions
     
+    /**
+     The access token used to make the directions request.
+     
+     Only set if a request is made via `Directions.calculate(_:completionHandler:)`.
```

Comment:
The current wording can be misread as, “You should only set this property if…”. How about:

> This property is set automatically if a request is made via `Directions.calculate(_:completionHandler:)`.

Same for `apiEndpoint` below.